cmd/migrate.go

```diff
@@ -0,0 +1,41 @@
+// Copyright the Service Broker Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////////
+//
+
+package cmd
+
+import (
+	"code.cloudfoundry.org/lager"
+	"github.com/GoogleCloudPlatform/gcp-service-broker/db_service"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(migrateCmd)
+}
+
+var migrateCmd = &cobra.Command{
+	Use:   "migrate",
+	Short: "Upgrade your database",
+	Long:  `Upgrade your database to be compatible with this service broker.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		logger := lager.NewLogger("migrations-cmd")
+		db := db_service.SetupDb(logger)
+		if err := db_service.RunMigrations(db); err != nil {
+			panic("Error running migrations")
```

Comment:
Consider adding a log.Fatal(err) instead of panic, as it will print information about the error to stdout and make it easier to debug issues.